### PR TITLE
Added new migration to force update for rsk legacy derivation path

### PR DESCRIPTION
**Pull request by bradleySuira**
Added new migration to force update for rsk legacy derivation path

## What?

https://www.notion.so/30ca4d49f3844a0fabdf4141f8a8bccc?v=44514edd52d14b719a9dec59d60147dd&p=900f7d3ace7d4d338fdf7a85306c09d9

### CODE REVIEW

File: src/store/migrations.js [hunk 1]
```diff
@@ -5,6 +5,22 @@ import { chains, assets as cryptoassets } from '@liquality/cryptoassets'
 import { v4 as uuidv4 } from 'uuid'
 import { shouldApplyRskLegacyDerivation } from './utils'
 
+// add a migration to reuse
+const rskLegacyDerivationMigration = async (state) => {
+  if (!Object.keys(state.accounts)?.length) {
```
Comment (harshjv) [inline src/store/migrations.js:10]: No need to use `?.length`. also, can you make it more descriptive with variable names?

**Review by harshjv — CHANGES_REQUESTED** [review]

File: src/store/migrations.js (hunk 1 above)
Comment (bradleySuira) [reply src/store/migrations.js:10]: oh ok, I just moved this from the old code, let me check now. thanks!

Comment (bradleySuira) [reply src/store/migrations.js]: done, fixed @harshjv 

BR!

**Review by harshjv — APPROVED** [review]
👍

**Pull request merged**